services/testService/package.json

```diff
@@ -1,26 +1,36 @@
 {
-  "name": "test_service",
-  "version": "1.2.3",
-  "description": "example microservice",
-  "keywords": [
-    "microservice"
-  ],
-  "main": "index.js",
-  "config": {
-    "services": {
-      "resourceConsumption": "singleton",
-      "state": "stateless",
-      "queue": true,
-      "role": {
+    "name": "test_service",
+    "version": "1.2.3",
+    "description": "example microservice",
+    "licence": "commercial",
+    "private": true,
+    "resourceConsumption": "normal",

```

Comment:
We will need to keep backward compatibility of the structure with a warning to tell them to move.
I agree to modify the structure, but why not keep all our "custom" properties in the same object ?

Reply:
I just thought it didn't really make sense since there are more properties that are custom to use:
dependantPackages
dependantServices
dependantFiles (git)
Unless we put these in the object too?

Reply:
Yes we can do.

Reply:
Done